### ISSUE TITLE
How do you create an arena? "/chickenset myarena [anything]" does absolutely nothing. 

### ISSUE THREAD

**Issue by ProGitHubUser** [nfacha/ChickenDefender#4]
Title is self explanatory.

**Comment by nfacha** [comment]
> Title is self explanatory.

At this time you have to clone the config section of the default area (and give it a different name), this will be added in a future update tho.

So if you wanted to have two arenas you would change your config from default as such:
```yml
config-version: 4 #DONT CHANGE THIS!
auto-update: true #Do we want to always keep the plugin updated?
update-channel: master #master = stable | dev = development versions
lobby: #General Lobby location, this is where players are teleported after the game os over (/chickenset lobby)
  x: 0
  y: 0
  z: 0
  world: world
arenas: #Config for each arenas, to add a new arena just duplicate the default item and give it a different name
  myarena:
    clear-inventory: true #Clear player inventory on join? (item are returned on leave) - Kits are not available if disabled
    team-helmet: false #Places a helmet on the player to help identify from what team he is - Only available if clear-inventory is enabled
    player-glow: true #Should the player glow with the color of his team? (This might always be white when some scoreboard plugins are present)
    default-kit: demo #Default kit given on join
    countdown: 30 #How long is the countdown after we have enough players?
    chicken-health: 50 #How many half hearts does the chicken have?
    player-lives: 3 #How many times do the players respawn before they are eliminated?
    players:
      min: 2 #Min players to trigger an arena start
      max: 10 #Max players allowed for this arena
    locations:
      world: world
      spawns:
        lobby: #Area lobby, where the player is after joining the arena and is waiting for the game to start (/chickenset arena 0 spawn)
          x: 0
          y: 0
          z: 0
        attacking: #Where does the attacking team spawn (/chickenset arena 0 attacking)
          x: 0
          y: 0
          z: 0
        defending: #Where does the defending team spawn (/chickenset arena 0 defending)
          x: 0
          y: 0
          z: 0
        chicken: #Where does the chicken spawn (/chickenset arena 0 chicken)
          x: 10
          y: 10
          z: 10
      corner1: #Top right corner of the arena region (/chickenset arena 0 corner1)
        x: 0
        y: 0
        z: 0
      corner2: #Bottom left corner of the arena region (/chickenset arena 0 corner2)
        x: 0
        y: 0
        z: 0
      sign: #Location of the arena join/status sign (/chickenset arena 0 sign)
        x: 0
        y: 0
        z: 0
        world: world
        enabled: false
        
  mysecondarena:
    clear-inventory: true #Clear player inventory on join? (item are returned on leave) - Kits are not available if disabled
    team-helmet: false #Places a helmet on the player to help identify from what team he is - Only available if clear-inventory is enabled
    player-glow: true #Should the player glow with the color of his team? (This might always be white when some scoreboard plugins are present)
    default-kit: demo #Default kit given on join
    countdown: 30 #How long is the countdown after we have enough players?
    chicken-health: 50 #How many half hearts does the chicken have?
    player-lives: 3 #How many times do the players respawn before they are eliminated?
    players:
      min: 2 #Min players to trigger an arena start
      max: 10 #Max players allowed for this arena
    locations:
      world: world
      spawns:
        lobby: #Area lobby, where the player is after joining the arena and is waiting for the game to start (/chickenset arena 0 spawn)
          x: 0
          y: 0
          z: 0
        attacking: #Where does the attacking team spawn (/chickenset arena 0 attacking)
          x: 0
          y: 0
          z: 0
        defending: #Where does the defending team spawn (/chickenset arena 0 defending)
          x: 0
          y: 0
          z: 0
        chicken: #Where does the chicken spawn (/chickenset arena 0 chicken)
          x: 10
          y: 10
          z: 10
      corner1: #Top right corner of the arena region (/chickenset arena 0 corner1)
        x: 0
        y: 0
        z: 0
      corner2: #Bottom left corner of the arena region (/chickenset arena 0 corner2)
        x: 0
        y: 0
        z: 0
      sign: #Location of the arena join/status sign (/chickenset arena 0 sign)
        x: 0
        y: 0
        z: 0
        world: world
        enabled: false

kits: #As the arenas each kit has its own config section, duplicate them as you need
  demo:
    name: Demo Kit 1 #Kit name displayed to the player
    items: #To add more items duplicate each section and increase the number
      0:
        type: DIAMOND_SWORD #Item type (possible types: https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/Material.html)
        name: Cute Sword #Item name
        amount: 1 #How much of this item?
        enchantments: #Each line its a new enchantment
          - FIRE_ASPECT-1 #Use the format "NAME-LEVEL" (possible names: https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/enchantments/Enchantment.html)
      1:
        type: WOODEN_SWORD
        name: Weak Sword
        amount: 1
        enchantments: [ ]
  demo2:
    name: Demo Kit 2
    items:
      0:
        type: DIAMOND_AXE
        name: Cute Axe
        amount: 1
        enchantments:
          - FIRE_ASPECT-1
```

I will leave this issue open until there is a way to create an arena thru commands.

**Comment by ProGitHubUser** [comment]
Ok. The commands still do nothing. The only command that gives ANY response at all is /chickenset lobby

**Comment by nfacha** [comment]
Just released version 1.3 (https://github.com/nfacha/ChickenDefender/releases/tag/1.3) that fixes this issue, you should use `/chickenset arena create NAME` and then follow the instructions to complete the setup and enable the arena.

Please note that you should take a look at the command list again, as the documentation was not updated and the `/chickenset` command was missing one parameter (so it did not reply at all)
